### PR TITLE
[Feature][Clickhouse-Connector-V2] Add Clickhouse e2e testcase

### PR DESCRIPTION
**Pull request by FWLamb**
## Purpose of this pull request

Add Clickhouse e2e testcase https://github.com/apache/incubator-seatunnel/issues/2949
## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Please fix ci error first, thanks.

**Comment by FWLamb** [comment]
Can't find some dependencies when building the ci process, but I can build successfully locally. How can I fix it?

**Comment by Hisoka-X** [comment]
> Can't find some dependencies when building the ci process, but I can build successfully locally. How can I fix it?

Network problem, I action it again.

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+    private ClickHouseContainer container;
+    private Connection connection;
+
+    @TestTemplate
+    public void testClickhouse(TestContainer container) throws Exception {
+        Container.ExecResult execResult = container.executeJob(CLICKHOUSE_JOB_CONFIG);
+        Assertions.assertEquals(0, execResult.getExitCode());
+        assertHasData(SINK_TABLE);
+        String columns = String.join(",", generateTestDataSet()._1().getFieldNames());
+        Assertions.assertTrue(compare(String.format(CONFIG.getString(COMPARE_SQL), columns, columns)));
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java]: could you check these

Check :  source table row count == sink table row count
Check:   source table row & column == sink table row & column

Comment (FWLamb) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java]: @Hisoka-X @hailin0 Sorry, operation error

Comment (FWLamb) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java]: > could you check these
> 
> Check : source table row count == sink table row count Check: source table row & column == sink table row & column

ok，I will check.

**Comment by FWLamb** [comment]
@Hisoka-X @hailin0 Please track it on https://github.com/apache/incubator-seatunnel/pull/3050

**Pull request closed without merging**